### PR TITLE
Updates to documentation

### PR DESCRIPTION
**Pull request by rubikzube**

### CODE REVIEW

File: AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift [hunk 1]
```diff
@@ -193,34 +197,34 @@ public class ConstructorIO: CIOSessionManagerDelegate {
...
+    /// Track when a user clicks an item that appears within a list of recommendation results
     ///
     /// - Parameters:
-    ///   - podID: Pod ID
-    ///   - customerID: Customer ID
-    ///   - strategyID: Strategy ID
-    ///   - variationID: Variation ID
-    ///   - numResultsPerPage: Count of recommendation results on each page
+    ///   - podID: The pod ID
+    ///   - strategyID: The strategy ID that fulfilled the po
```
Comment (crgee1) [inline AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift:221]: typo?

File: AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift [hunk 2]
```diff
@@ -193,34 +197,34 @@ public class ConstructorIO: CIOSessionManagerDelegate {
...
+    ///   - customerID: The item ID
+    ///   - variationID: The item variation I
+    ///   - numResultsPerPage: The count of recommendation results on each pag
```
Comment (crgee1) [inline AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift:224]: typo?

File: AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift [hunk 3]
```diff
@@ -193,34 +197,34 @@ public class ConstructorIO: CIOSessionManagerDelegate {
...
     /// - Parameters:
-    ///   - podID: Pod ID
-    ///   - customerID: Customer ID
-    ///   - strategyID: Strategy ID
-    ///   - variationID: Variation ID
-    ///   - numResultsPerPage: Count of recommendation results on each page
+    ///   - podID: The pod ID
+    ///   - strategyID: The strategy ID that fulfilled the po
+    ///   - customerID: The item ID
+    ///   - variationID: The item variation I
```
Comment (crgee1) [inline AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift:223]: typo?

File: AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift [hunk 4]
```diff
@@ -193,34 +197,34 @@ public class ConstructorIO: CIOSessionManagerDelegate {
...
+    ///   - numResultsPerPage: The count of recommendation results on each pag
     ///   - resultPage: The current page that recommedantion result is on
     ///   - resultCount: The total number of recommendation results
-    ///   - resultPositionOnPage: The position of the recommendation result that was clicked on
+    ///   - resultPositionOnPage: The position of the recommendation result that was clicked o
```
Comment (crgee1) [inline AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift:227]: typo?

**Review by crgee1 — COMMENTED** [review]
Looks good, just saw a couple of potential typos I left comments on

**Review by jjl014 — APPROVED** [review]
Looks good to me. 👍 

Not sure how much it matters, but do we want to line up the order of the documented params to the order of the function params?

EDIT: not sure how i didn't notice the typos, good catch @crgee1!

**Pull request merged**